install.sh

```diff
@@ -1037,7 +1037,26 @@ download_latest_bundle_and_extract_if_needed() {
     rm -rf $BUILD_DIR
     local URL="https://dl.sandstorm.io/sandstorm-$BUILD.tar.xz"
     echo "Downloading: $URL"
-    curl -A "$CURL_USER_AGENT" -f "$URL" | tar Jxo
+    curl -A "$CURL_USER_AGENT" -f "$URL" > /var/tmp/sandstorm-$BUILD.tar.xz
+    curl -s -A "$CURL_USER_AGENT" -f "$URL.sig" > /var/tmp/sandstorm-$BUILD.tar.xz.sig
+    curl -s -A "$CURL_USER_AGENT" -f "https://dl.sandstorm.io/keyring.gpg" > /var/tmp/sandstorm-keyring.gpg
+
+    if which gpg > /dev/null; then
+      if gpg --no-default-keyring --keyring /var/tmp/sandstorm-keyring.gpg --status-fd 1 \
```

Comment:
Yes, you can do the "append arbitrary data" attack against a detached sig if you convert it to an inline sig, as far as I can tell.

But it _looks_ like gpg will insist on a detached signature in the two-argument `gpg --verify x.sig x` invocation:


```
[GNUPG:] NODATA 1
gpg: not a detached signature
```

So I guess this is reasonable. `man gpg` agrees. so I'm +1.

But if we know anyone who's more of a GPG expert than me, I'd be very happy to get their review on this. (e.g. @diracdeltas?) But I'm also OK with shipping this as-is, and flagging that for being done "eventually".